File: C#/RiceStore/RiceStore/Staff.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RiceStore
{
    public class Staff
    {
        private string id;
        private string name;
        private string workShift;
        private string address;
        private bool gender;
        private DateTime birthday;
        private string phone;

        public Staff(string id, string name, string workShift, string address, bool gender, DateTime birthday, string phone)
        {
            this.id = id;
            this.name = name;
            this.workShift = workShift;
            this.address = address;
            this.gender = gender;
            this.birthday = birthday;
            this.phone = phone;
        }

        public string getId()
        {
            return id;
        }
        public void setId(string id)
        {
            this.id = id;
        }
        public string getName()
        {
               return name;
        }
        public void setName(string name)
        {
            this.name = name;
        }
        public string getWorkShift()
        {
            return workShift;
        }
        public void setWorkShift(string workShift)
        {
            this.workShift = workShift;
        }
        public string getAddress()
        {
            return address;
        }
        public void setAddress(string address)
        {
            this.address = address;
        }
        public bool getGender()
        {
            return gender;
        }
        public void setGender(bool gender)
        { 
            this.gender = gender;
        }
        public DateTime getBirthday()
        {
            return birthday;
        }
        public void setBirthday(DateTime birthday)
        {
            this.birthday = birthday;
        }
        public string getPhone()
        {
            return phone;
        }
        public void setPhone(string phone)
        {
            this.phone = phone;
        }
    }
}

File: Java/RiceStore/src/Model/Staff.java
package Model;
import java.util.Date;

public class Staff {
    private String id;
    private String name;
    private String workShift;
    private String address;
    private Boolean gender;
    private Date birthday;
    private String phone;

    public Staff(String id, String name, String workShift, String address, Boolean gender, Date birthday, String phone) {
        this.id = id;
        this.name = name;
        this.workShift = workShift;
        this.address = address;
        this.gender = gender;
        this.birthday = birthday;
        this.phone = phone;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getWorkShift() {
        return workShift;
    }

    public void setWorkShift(String workShift) {
        this.workShift = workShift;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public Boolean getGender() {
        return gender;
    }

    public void setGender(Boolean gender) {
        this.gender = gender;
    }

    public Date getBirthday() {
        return birthday;
    }

    public void setBirthday(Date birthday) {
        this.birthday = birthday;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }
}
